### ISSUE TITLE
Debian/Ubuntu with eBPF package fails to build correctly due to shlibdeps

### ISSUE THREAD

**Issue by prologic** [netdata/netdata#9550]
See: https://travis-ci.com/github/netdata/netdata/jobs/361530214

**Comment by prologic** [comment]
In fact we seem to have zero Debian packages.

See: https://travis-ci.com/github/netdata/netdata/jobs/361530329

They're all failing with Debian's library checking tools on libelf and various other libraries from the pre-built eBPF shared libraries.

**Comment by Steve8291** [comment]
Same problem for me on Ubuntu 18.04.4
eBPF is running and creating graphs for my netdata site but the installer fails and leaves the temporary directory it created from line 714 of the `netdata-installer.sh`
I get the following messages during an install:
```
v0.0.9.tar.gz: OK
[/tmp/netdata-kickstart-EDpSPN/netdata-v1.23.2-15-g176b6e5f]# tar -xf /tmp/netdata-libbpf-OoE9N5/v0.0.9.tar.gz -C /tmp/netdata-libbpf-OoE9N5 
 OK   

[/tmp/netdata-kickstart-EDpSPN/netdata-v1.23.2-15-g176b6e5f]# git apply --directory=/tmp/netdata-libbpf-OoE9N5/libbpf-0.0.9 --unsafe-paths libnetdata/ebpf/libbpf.c.diff 
error: can't open patch 'libnetdata/ebpf/libbpf.c.diff': No such file or directory
 FAILED   

 FAILED  Failed to build libbpf. 
```

**Comment by thiagoftsm** [comment]
Hi @Steve8291 , 

we are changing the way Netdata loads the ebpf.plugin and probably we are missing something on Ubuntu.
We have the missing file inside our branch, but I think we are missing something when `kickstart` is used.
@Ferroin, @prologic do you know what can be the problem with `kickstart`?

**Comment by prologic** [comment]
@Ferroin Fixed this in #9628 and we know why this failed and now we have CI checks
